package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "maintenance-tracker",
+  "version": "1.0.0",
+  "description": "Helps to track maintenance requests of user",
+  "main": "index.js",
+  "scripts": {
+    "build": "babel lib -d dist",
+    "pretest": "eslint --ignore-path .gitignore .",
```

Comment:
what is the purpose of doing this ,can you not just test a specific folder ?

Reply:
Currently, I have my folder structure is this way 👇 

![image](https://user-images.githubusercontent.com/8234355/40285245-f1e0a514-5c91-11e8-9a74-635241f27ffd.png)

I want eslint to check all files in each of the folders, except for `node_modules` and `dist` and both files are also ignored by git, so they are in my `.gitignore`, that was why I did the `--ignore-path .gitignore`.

I do not know how testing just a specific folder will work since I need to test more than one folder at this point, unless I restructure the project and put all those in 1 folder, Do you advice that?

Reply:
I have refactored my code a little bit, I am currently testing the `app` and `lib` folder